### PR TITLE
Fix Admin Screen Headings/Hierarchy

### LINKED ISSUE #10300: Admin screen heading tags with WordPress 4.4
Woocommerce uses incorrect headings in admin screens. See https://make.wordpress.org/core/2015/10/28/headings-hierarchy-changes-in-the-admin-screens/ for more.

### PR DESCRIPTION
**Pull request by emdashcodes**
Fixes #10300.

This PR brings all of the core WooCommerce admin screens inline with the recommendations for heading hierarchy mentioned at https://make.wordpress.org/core/2015/10/28/headings-hierarchy-changes-in-the-admin-screens/ including: Top level headings for all screen (even if just for screen readers), no skipped headings, and fixed screen reader text for list tables.

### CODE REVIEW

File: includes/admin/views/html-admin-page-reports.php [hunk 1]
```diff
@@ -52,8 +56,12 @@
 
 		$report = $reports[ $current_tab ][ 'reports' ][ $current_report ];
 
-		if ( ! isset( $report['hide_title'] ) || $report['hide_title'] != true )
-			echo '<h3>' . $report['title'] . '</h3>';
+		if ( ! isset( $report['hide_title'] ) || $report['hide_title'] != true ) {
+			echo '<h2>' . esc_html( $report['title'] ) . '</h2>';
+		}
+		else {
```
Comment (mikejolley) [inline includes/admin/views/html-admin-page-reports.php]: else on same line as brace https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#brace-style

File: assets/css/admin.scss [hunk 2]
```diff
@@ -3903,7 +3913,7 @@ img.ui-datepicker-trigger {
 			margin: 0 !important;
 		}
 
-		h3.stats_range {
+		div.stats_range,h3.stats_range {
```
Comment (mikejolley) [inline assets/css/admin.scss]: Looks like not done up until now but technically each selector should be on its own line. I usually at least space them https://make.wordpress.org/core/handbook/best-practices/coding-standards/css/#structure

**Comment by mikejolley** [comment]
LGTM, added minor comment. Also needs syncing upstream.

**Comment by claudiosmweb** [comment]
LGTM for me too

**Pull request merged**